ui/raidboss/data/06-ew/raid/p9s.txt

```diff
@@ -29,20 +46,178 @@ hideall "--sync--"
+94.1 "Pile Pyre" #sync / 1[56]:[^:]*:Kokytos:(8156|8158):/
 
 101.8 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8124:/
 
-# Fighter 1
-114.2 "Ravening (Fighter 1)" sync / 1[56]:[^:]*:Kokytos:8119:/ window 40,40
+# Fighter
+114.2 "Ravening (Fighter)" sync / 1[56]:[^:]*:Kokytos:8119:/ window 40,40
 120.9 "Soul Surge" sync / 1[56]:[^:]*:Kokytos:811B:/
+134.0 "Ascendant Fist" sync / 1[56]:[^:]*:Kokytos:816F:/
+
+146.4 "Archaic Rockbreaker (cast)" sync / 1[56]:[^:]*:Kokytos:815F:/
+148.0 "Shockwave" sync / 1[56]:[^:]*:Kokytos:8160:/
+149.3 "Archaic Rockbreaker (resolve)" sync / 1[56]:[^:]*:Kokytos:8162:/
+155.5 "Front Combination/Rear Combination" sync / 1[56]:[^:]*:Kokytos:(8167|8168|8169|816A):/
+155.8 "Archaic Rockbreaker #1" sync / 1[56]:[^:]*:Kokytos:8161:/
+156.3 "Inside Roundhouse/Outside Roundhouse" sync / 1[56]:[^:]*:Kokytos:(8238|8239):/
+158.7 "Swinging Kick" sync / 1[56]:[^:]*:Kokytos:(816B|816C):/
+161.7 "Archaic Rockbreaker #2" sync / 1[56]:[^:]*:Kokytos:8161:/
+162.4 "Outside Roundhouse/Inside Roundhouse" sync / 1[56]:[^:]*:Kokytos:(8239|8238):/
+168.7 "Archaic Demolish" sync / 1[56]:[^:]*:Kokytos:816D:/
+
+175.8 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:814C:/
+187.1 "Ascendant Fist" sync / 1[56]:[^:]*:Kokytos:816F:/
+
+194.5 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8125:/
+
+# Chimera 1
+202.9 "--middle--" sync / 1[56]:[^:]*:Kokytos:8144:/
+208.2 "Ravening (Chimera #1)" sync / 1[56]:[^:]*:Kokytos:817B:/
+214.8 "Soul Surge" sync / 1[56]:[^:]*:Kokytos:811B:/
+
+221.4 "Levinstrike Summoning" sync / 1[56]:[^:]*:Kokytos:817C:/
+233.5 "Scrambled Succession" sync / 1[56]:[^:]*:Kokytos:817D:/
+233.6 "--untargetable--"
+
+# Limit Cut 1
+233.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+235.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:817E:/
+238.4 "Firemeld #1" sync / 1[56]:[^:]*:Kokytos:8180:/
+238.5 "Icemeld #1" sync / 1[56]:[^:]*:Kokytos's Echo:8183:/
+239.4 "Shock #1" sync / 1[56]:[^:]*:Kokytos:8181:/
+
+239.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+241.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:817E:/
+244.1 "Firemeld #2" sync / 1[56]:[^:]*:Kokytos:8180:/
+244.3 "Icemeld #2" sync / 1[56]:[^:]*:Kokytos's Echo:8190:/
+245.2 "Shock #2" sync / 1[56]:[^:]*:Kokytos:8181:/
+
+245.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+246.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:817E:/
+249.7 "Firemeld #3" sync / 1[56]:[^:]*:Kokytos:8180:/
+250.0 "Icemeld #3" sync / 1[56]:[^:]*:Kokytos's Echo:8191:/
+250.7 "Shock #3" sync / 1[56]:[^:]*:Kokytos:8181:/
+
+250.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+252.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:817E:/
+255.4 "Firemeld #4" sync / 1[56]:[^:]*:Kokytos:8180:/
+255.8 "Icemeld #4" sync / 1[56]:[^:]*:Kokytos's Echo:8192:/
+256.5 "Shock #4" sync / 1[56]:[^:]*:Kokytos:8181:/
+256.5 "--middle--" #sync / 1[56]:[^:]*:Kokytos:8144:/
+
+258.1 "--targetable--"
+262.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:(8122|8123|815C):/
+265.1 "Two Minds" sync / 1[56]:[^:]*:Kokytos:(8184|8185):/
```

Comment:
If you're at all interested in followup trigger PRs, I didn't have time to figure out the two ability ids here in terms of which is which, and they could be added to the ids for the dualspell triggers to get better callouts for them.

Reply:
I'll circle back to it if I get some free time in the next week or two.